content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -297,11 +297,109 @@ One use case for selecting over labels is to constrain the set of nodes onto whi
 a pod can schedule. See the documentation on
 [node selection](/docs/concepts/scheduling-eviction/assign-pod-node/) for more information.
 
+## Using labels effectively
+
+You can apply a single label to any resources, but this is not always the
+best practice. There are many scenarios where multiple labels should be used to
+distinguish resource sets from one another.
+
+For instance, different applications would use different values for the `app` label, but a
+multi-tier application, such as the [guestbook example](https://github.com/kubernetes/examples/tree/master/guestbook/),
+would additionally need to distinguish each tier. The frontend could carry the following labels:
+
+```yaml
+labels:
+  app: guestbook
+  tier: frontend
+```
+
+while the Redis master and slave would have different `tier` labels, and perhaps even an
```

Comment:
Eeesh. Some **other** PR should avoid the terms “master“ and “slave“; some countries still permit slavery or even officially endorse it; other places merely regret their history of involvement.

There are actual victims of slavery who find this usage unsettling.